### ISSUE TITLE
Starting `emacsclient -c` makes EXWM lose control of all EXWM buffers

### ISSUE THREAD

**Issue by Ambrevar** [ch11ng/exwm#305]
When you have some EXWM buffers open, start `emacsclient -c`: all the EXWM buffers are gone, but the associated programs are still running in the background.

I also had the same issue with some program involving `server-edit`, like the commit message of `phabrikator`, but I'm not quite sure of what triggers the issue in that case.

**Comment by ch11ng** [comment]
Creating an emacsclient frame just adds another workspace.  How to reproduce this?

**Comment by Ambrevar** [comment]
Oops! Did not notice...
Alright, so there is no bug here.  But that raises another point: is there any indicator of which workspace the user is currently on?

**Comment by ch11ng** [comment]
The user can invoke <kbd>M-x exwm-workspace-switch</kbd> to tell which is the current workspace (and <kbd>C-g</kbd> afterwards).

**Comment by Ambrevar** [comment]
I think this highlights a usage flaw: By starting `emacsclient -c`, or, in my case, a script calling it unbeknownst to me, I end up with all my EXWM windows gone (because of the workspace switch) without any hint of what happened.  This confused me terribly.

My suggestion: When there is more than one active workspace, display it in the modeline, e.g. `[3/5]` for workspace number 3 out of 5 active workspaces.

**Comment by ch11ng** [comment]
So it need to be displayed in all buffers' modeline I guess.

**Comment by Ambrevar** [comment]
Yup.  If you think some users would dislike that EXWM clutters the modeline, make this an option.
